### PR TITLE
Weather app using API

### PR DESCRIPTION
**Pull request by NiiazalyDzhumaliev**
The project was created using API:

- fetch data;
- async/await;
- code refactored into modules;
- basic styling using Bootstrap.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,69 @@
...
+
+### Install
+
+```
+- git clone https://github.com/paulzay/weatherapp;
+- cd weatherapp/
+- npm install
+- npm webpack --watch
+```
+
```
Comment (christianotieno) [inline README.md:40]: ## Add a live link to the page

- [ ] Add a live link to where the page is live. let's make sure the README is more descriptive. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

File: package.json [hunk 2]
```diff
@@ -17,6 +17,9 @@
   },
   "homepage": "https://github.com/NiiazalyDzhumaliev/weatherApp#readme",
   "devDependencies": {
+    "css-loader": "^5.0.0",
+    "file-loader": "^6.1.1",
+    "style-loader": "^2.0.0",
     "webpack": "^5.1.3",
     "webpack-cli": "^4.1.0"
   }
```
Comment (christianotieno) [inline package.json:27]: ## Webpack configuration

- [ ] All webpack related packages should be included in `package.json` like `webpack`, `webpack-cli`, `webpack-dev-server` etc. Also, those packages should be added as **devDependencies.** We are currently missing some of these in this file, let's add them. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/devdep_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))

- [ ] We need to have necessary run common commands like `build`, `development` to be present in `package.json` inside \`scripts'. Here is a basic example we can draw inspiration from 👇 ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))

```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

**Review by christianotieno — CHANGES_REQUESTED** [review]
### Changes Required ♻️ 
Hello @NiiazalyDzhumaliev 👋 
You have done a good job so far. Keep it up. 👏 
Kindly take care of the following highlighted issues before we proceed. 👇 

### Happy coding 💻 

#### Christian Otieno (TSE)

**Comment by NiiazalyDzhumaliev** [comment]
Thanks for the reasonable comments.

- add a live version of the app;
- add the related packages;
- add scripts.

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @NiiazalyDzhumaliev 
### You have done well so far :clap: However, you have one issue to fix:-
- [ ] Add a toggle button to determine if the result will be shown in Celsius or Fahrenheit
### Make the change and submit for another code review. Good luck.
- Happy coding!

**Review by rOluochKe — CHANGES_REQUESTED** [review]
### Make the changes from the previous review and submit for another code review.

**Comment by NiiazalyDzhumaliev** [comment]
Add toggle button to switch degree measures.

**Review by whiz25 — APPROVED** [review]
## Approved :clap: 
Nice work addressing the changes requested. Merge and proceed to the next project.


### Martin Afani

**Pull request merged**